homeassistant/components/ziggo_next/media_player.py

```diff
@@ -0,0 +1,183 @@
+"""Support for interface with a Ziggo Mediabox Next."""
+import logging
+import random
+
+from ziggonext import ONLINE_RUNNING, ONLINE_STANDBY, ZiggoNext
+
+from homeassistant.components.media_player import MediaPlayerDevice
+from homeassistant.components.media_player.const import (
+    MEDIA_TYPE_APP,
+    MEDIA_TYPE_CHANNEL,
+    SUPPORT_NEXT_TRACK,
+    SUPPORT_PAUSE,
+    SUPPORT_PLAY,
+    SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_SELECT_SOURCE,
+    SUPPORT_TURN_OFF,
+    SUPPORT_TURN_ON,
+)
+from homeassistant.const import (
+    STATE_OFF,
+    STATE_PAUSED,
+    STATE_PLAYING,
+    STATE_UNAVAILABLE,
+)
+
+from .const import DOMAIN, ZIGGO_API
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_devices):
+    """Set up entry."""
+    players = []
+    api = hass.data[ZIGGO_API]
+    for box_id in api.settopBoxes.keys():
```

Comment:
You could use `for box_id, box in api.settopBoxes.items()` here. If you control the backend library, `settop_boxes` would be more a pythonic naming for the variable.